.github/test-spec.yml

```diff
@@ -84,7 +84,20 @@
   - "**/*"
 
 "CI-matter-test":
-  - "**/*"
+  - "include/dfu/**/*"
+  - "include/mgmt/mcumgr/**/*"
+  - "soc/arm/nordic_nrf/**/*"
+  - "subsys/dfu/**/*"
+  - "subsys/settings/**/*"
+  - "subsys/net/**/*"
+  - "subsys/mgmt/mcumgr/**/*"
+  - "drivers/net/**/*"
+  - "drivers/wifi/**/*"
+  - "samples/hci_rpmsg/**/*"
```

Comment:
Are these samples entries needed? Do our tests depend on mcumgr samples?

Reply:
Do you mean hci_rpmsg?
We use that in wifi sample.

Regarding next record mcumgr, I thought we were basing our implementation of SMP support on this example

Reply:
We use SMP server, but not the sample, so this shouldn't affect us, I think.

Reply:
ok, removed